### PR TITLE
Fixed : Team section , Mentor Section , Navigation of FAQ

### PR DESCRIPTION
**Pull request by Sheetal-04**

### CODE REVIEW

File: src/Components/hero/index.jsx [hunk 1]
```diff
@@ -40,8 +40,8 @@ export default function Hero() {
             <div className="lg:grid lg:grid-cols-2 lg:gap-8">
               <div className="mx-auto max-w-md px-4 sm:max-w-2xl sm:px-6 sm:text-center lg:px-0 lg:text-left lg:flex lg:items-center">
                 <div className="lg:py-24">
-                  <div className="flex justify-center items-center text-center dark:bg-black shadow-md dark:text-white text-xs font-normal rounded-lg px-2 py-1 w-fit mx-0 sm:mx-auto lg:mx-0">
-                    Your Gateway to Infinite Opportunities
+                  <div className="flex justify-center items-center text-center dark:bg-black shadow-md dark:text-white text-[16px] font-normal rounded-lg px-2 py-1 w-fit mx-0 sm:mx-auto lg:mx-0">
```
Comment (Nandani-Paliwal) [inline src/Components/hero/index.jsx]: use `text-base` in place of `text-[16px]`

File: src/Components/mentors-word/index.jsx [hunk 2]
```diff
@@ -42,10 +42,10 @@ export default function MentorsWords() {
                     </p>
                   </div>
                   <footer className="mt-6">
-                    <p className="text-base font-medium text-black dark:text-primary-500">
+                    <p className="text-lg font-bold text-primary-900 dark:text-primary-500">
```
Comment (Nandani-Paliwal) [inline src/Components/mentors-word/index.jsx]: change the color from `primary-900` to `text-white` same as opportunityNexus

**Review by Nandani-Paliwal — CHANGES_REQUESTED** [review]

File: src/Components/team/index.jsx [hunk 3]
```diff
@@ -27,19 +30,17 @@ const people = [
...
+    linkedinUrl: "https://www.linkedin.com/in/techyvishwajeet/",
   },
 
   {
     name: "Prashant Dubey",
     role: "Copywriter",
-    imageUrl:
-      "https://images.unsplash.com/photo-1517365830460-955ce3ccd263?ixlib=rb-=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=facearea&facepad=8&w=1024&h=1024&q=80",
+    imageUrl:PrashantAvatar,
     twitterUrl: "#",
```
Comment (Nandani-Paliwal) [inline src/Components/team/index.jsx:42]: remove it if he doesn't have a twitter account

**Review by Nandani-Paliwal — DISMISSED** [review]

**Comment by Nandani-Paliwal** [comment]
@Sheetal-04  pull the changes of `main` branch into `minor-fixes`

**Review by AmanGupta2626 — APPROVED** [review]

**Pull request merged**